operations/helm/charts/mimir-distributed/templates/alertmanager/alertmanager-statefulset.yaml

```diff
@@ -53,11 +53,20 @@ spec:
     metadata:
       labels:
         {{- include "mimir.podLabels" $args | nindent 8 }}
+        grafana.com/no-downscale: {{$rolloutZone.noDownscale}}
```

Comment:
I don't think this is needed for the alertmanager, I'm pretty sure they're safe to scale down at any time. Could one of the alerting folks confirm? @stevesg et al